docs/reference/technology/mesh/thread_faq.md

```diff
@@ -49,7 +49,7 @@ Link quality and errors are easiest to debug using Wireshark.
 
 #### How to send/receive with UDP and TCP sockets. How to use multicast groups?
 
-The mesh minimal uses socket communication and multicasting. See more details in the [mesh minimal example](https://github.com/ARMmbed/mbed-os-example-mesh-minimal).
+The mesh minimal uses socket communication and multicasting. See more details in the [mesh minimal example](../apis/mesh-api.html#mesh-example).
```

Comment:
Socket communication is not documented in https://os.mbed.com/docs/v5.10/apis/mesh-api.html#mesh-example. This is because file https://github.com/ARMmbed/mbed-os-example-mesh-minimal/blob/master/mesh_led_control_example.cpp is missing from the example.  Can the example in docs be updated to contain all files or should the link point to the original application in github?

Reply:
Does this version include the updates? https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-mesh-minimal/file/c841ae60db06/main.cpp/

Reply:
Yes it does, the file https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-mesh-minimal/file/c841ae60db06/mesh_led_control_example.cpp/ is available there. 
I was a little bit confused how to find file `mesh_led_control_example.cpp` as it is not available in the https://os.mbed.com/docs/v5.11/apis/mesh-api.html#mesh-example. 
Maybe file `mesh_led_control_example.cpp` could be added also to the https://os.mbed.com/docs/v5.11/apis/mesh-api.html#mesh-example if it is not confusing the user too much?

Reply:
PR up: https://github.com/ARMmbed/mbed-os-5-docs/pull/889